docs/dev-proc/hcl-maintainer.md

```diff
@@ -7,18 +7,31 @@ Compatibility List for CPUs, memory, mainboards and GPU.
 
 ### Dasharo HCL report parsing
 
-For reports uploaded to 3mdeb cloud please use [dedicated
-script](https://github.com/Dasharo/dts-hw-conf-gen) readme.
+Use [dts-hw-conf-gen](https://github.com/Dasharo/dts-hw-conf-gen) tool for
+automatic HCL report parsing.
+
+Reports can be sourced from:
+- 3mdeb Cloud
+- Minio
```

Comment:
Nah, I'm pretty sure we have a better name for that service, @artur-rs ?

Reply:
@pietrushnic @EduKav1813 previously we have used:
* `DPP packaging service` - https://github.com/Dasharo/meta-dts/releases/tag/v2.0.0
* `DPP download page` - https://docs.dasharo.com/osf-trivia-list/dasharo-pro-package/

We could also use: `DPP sharing service`, `DPP MinIO`...

Reply:
```suggestion
- DPP download page
```